R/functions.R

```diff
@@ -1,12 +1,14 @@
 #' Sum numeric columns of input data
 #' @param data dataset on which to perform operation
+#' @export
 sum_columns <- function(data){
```

Comment:
Based on http://adv-r.had.co.nz/Style.html we should add space before `{` sign:
`function(data){` -> `function(data) {`

Please fix below function definitions as well.

Reply:
fixed